CONTRIBUTING.md

```diff
@@ -187,4 +187,18 @@ go vet
 Unit tests for the Feast Go Client can be run as follows:
 ```sh
 go test
-```
\ No newline at end of file
+```
+
+### Testing with Github Actions workflows
+* Update your current master on your forked branch and make a pull request against your own forked master.
+* Enable workflows by going to actions and clicking `Enable Workflows`.
+    * Pushes will now run your edited workflow yaml file against your test code.
+    * Unfortunately, in order to test any github workflow changes, you must push the code to the branch and see the output in the actions tab.
+
+## Issues
+* pr-integration-tests workflow is skipped
+    * Temporarily remove the `on:` block at the top and replace with `on: [push, pull_request]`.
```

Comment:
I don't remember seeing this error when I did this, so I suspect there's a way around this. I'll poke around; let's hold this PR for now.